CODEOWNERS

```diff
@@ -463,6 +463,7 @@ build.json @home-assistant/supervisor
 /tests/components/http/ @home-assistant/core
 /homeassistant/components/huawei_lte/ @scop @fphammerle
 /tests/components/huawei_lte/ @scop @fphammerle
+/homeassistant/components/huawei_router/ @abmantis
```

Comment:
```suggestion
```

Reply:
Thanks, I wanted it to be a single commit though. I've got it fixed and pushed the correct one.

Reply:
My bad, sorry. Noted for the future.